docs/installation.rst

```diff
@@ -0,0 +1,139 @@
+.. _installation:
+
+************
+Installation
+************
+
+Installing ``hvpy``
+-------------------
+
+If you are new to Python and/or do not have familiarity with `Python virtual environments <https://docs.python.org/3/tutorial/venv.html>`__, then we recommend starting by installing `mini-forge <https://github.com/conda-forge/miniforge#miniforge3>`__.
+This works on all platforms (Linux, Mac, Windows) and installs a full-featured Python environment in your user directory without requiring root privileges.
+
+Using pip
+---------
+
+.. warning::
+
+    Users of any conda distribution should follow the instructions for :ref:`conda_install`.
+
+To install ``hvpy`` with `pip <https://pip.pypa.io/en/stable/>`__, run ::
+
+    pip install hvpy
+
+.. _conda_install:
+
+Using Conda
+-----------
+
+To install ``hvpy`` using `conda <https://docs.conda.io/projects/conda/en/latest/>`__ run ::
+
+    conda install -c conda-forge hvpy
+
+Obtaining the Source Package
+----------------------------
+
+Source Package
+^^^^^^^^^^^^^^
+
+The latest stable source package for ``hvpy`` can be `downloaded at pypi.org <https://pypi.org/project/hvpy>`__.
+
+Development Repository
+^^^^^^^^^^^^^^^^^^^^^^
+
+The latest development version of ``hvpy`` can be cloned from `GitHub <https://github.com/Helioviewer-Project/python-api/>`__ ::
+
+   git clone git@github.com:Helioviewer-Project/python-api.git
+
+Building and Installing
+^^^^^^^^^^^^^^^^^^^^^^^
+
+To build and install ``hvpy`` (from the root of the source tree) ::
+
+    pip install -e .
+
+which installs ``hvpy`` in development mode, this then means that changes in the code are immediately reflected in the installed version.
+
+Building Documentation
+----------------------
+
+.. note::
+
+    Building the documentation is in general not necessary unless you are
+    writing new documentation or do not have internet access, because
+    the latest versions of hvpy's documentation should be available
+    at `hvpy.readthedocs.io <https://hvpy.readthedocs.io/>`_ .
+
+Dependencies
+^^^^^^^^^^^^
+
+Building the documentation requires the ``hvpy`` source code and some
+additional packages. The easiest way to build the documentation is to use `tox
+<https://tox.readthedocs.io/en/latest/>`_ as detailed in
+:ref:`hvpy-doc-building`.
```

Comment:
```suggestion
Building the documentation requires the ``hvpy`` source code and some additional packages.
The easiest way to build the documentation is to use `tox <https://tox.readthedocs.io/en/latest/>`__ as detailed in :ref:`hvpy-doc-building`.
```

Reply:
You want to do 1 line per sentence instead of breaking up text into random length lines.